### PR TITLE
Cloak geo input polygon enable

### PR DESCRIPTION
**Pull request by AdityaKhatri**

### CODE REVIEW

File: src/components/input/GeoInput/index.js [hunk 1]
```diff
@@ -202,6 +216,10 @@ export default class GeoInput extends React.PureComponent {
         const polygons = this.getPolygons(value);
         const options = this.getAllGeoOptions(geoOptionsByRegion);
 
+        const polygonSupportIndex = accessibleFeatures.findIndex(f => f.key === 'polygon_support_geo');
```
Comment (tnagorra) [inline src/components/input/GeoInput/index.js]: We should create constants for feature name somewhere

File: src/components/input/GeoInput/index.js [hunk 2]
```diff
@@ -202,6 +216,10 @@ export default class GeoInput extends React.PureComponent {
...
+        const isPolygonFeatureEnabled = polygonSupportIndex !== -1;
+        const shouldEnablePloygon = isPolygonFeatureEnabled && polygonsEnabled;
```
Comment (tnagorra) [inline src/components/input/GeoInput/index.js]: Spelling mistake for "Polygon"
Try using this plugin for vim https://github.com/kamykn/spelunker.vim

**Review by tnagorra — CHANGES_REQUESTED** [review]
Need to create accessible feature constants

File: src/components/input/GeoInput/index.js (hunk 1 above)
Comment (AdityaKhatri) [reply src/components/input/GeoInput/index.js]: yes we should

Comment (AdityaKhatri) [reply src/components/input/GeoInput/index.js]: In constants?

File: src/constants/features.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/prefer-default-export
+const featuresMapping = {
+    polygonSupportGeo: 'polygon_support_geo',
+    zoomableImage: 'zoomable_image',
+    privateProject: 'privateProject',
```
Comment (tnagorra) [inline src/constants/features.js]: The feature names from server are usually snake_case. This looks problematic

Comment (AdityaKhatri) [reply src/constants/features.js]: It was a test. You passed!

**Review by tnagorra — APPROVED** [review]

**Pull request merged**